### ISSUE TITLE
Accessibility: unable to access actions while using talkback

### ISSUE THREAD

**Issue by shortwavesurfer2009** [boshi-chan/nostr-monstr#4]
I am unable to access actions when using talkback such as reply, zap, boost, etc. I can see that they are red, but they are not actual actions that I can use.

Example of talkback output:

"boseph boseph @boseph@monstr.land Verified Post options just now Following Replying to @boseph @shortwavesurfer2009 I'll admit, global is a useless firehose, I doubt it'll be around much longer, I never use it Like Comment Repost options Zap Ember"
